packages/api-derive/src/balances/fees.ts

```diff
@@ -22,25 +20,30 @@ const ZERO = new BN(0);
  *
  * ```javascript
  * api.derive.balances.fees(([creationFee, transferFee]) => {
- *   console.log(`The fee for creating a new account on this chain is ${creationFee} units. The fee required for making a transfer is ${transferFee} units.`);
+ *   // console.log(`The fee for creating a new account on this chain is ${creationFee} units. The fee required for making a transfer is ${transferFee} units.`);
```

Comment:
Why is this commented?

Reply:
Oh, a leftover from development. I commented all console.logs to do less scrolling when comparing the results of different nodes. So an oversight.

Reply:
Phew, was really wondering about that one :)